github-page/_guides/migrate-to-6.md

```diff
@@ -45,13 +45,14 @@ otherwise the functions still override the settings made in `modelOptions`
 Hooks got (in 6.0.0-13) a change for the types to comply with the latest mongoose (5.6.8)
 -> no workarounds or typedefs required anymore
 
-## Methods (staticMethod, instanceMethod, virtuals)
+## Methods (static, instance, virtuals)
```

Comment:
it had an reason why it was `staticMethod, instanceMethod`

Reply:
Yeah, but for that reason, searching for "static" doesn't find anything.

Reply:
The old `@staticMethod` & `@instanceMethod` are still there.

Reply:
i know that the documentation is still missing some parts, like how to use functions (because i hear that often)